release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -1298,6 +1363,8 @@ $ rm -f openshift/99_openshift-cluster-api_worker-machineset-*.yaml
 * Currently, the prerequisites in the web console quick start cards appear as a paragraph instead of a list. This will be resolved in a future release. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1905147[BZ#1905147])
 * Currently, in the *Search Page*, the *Pipelines* resources table is not immediately updated after the *Name* filter is applied or removed. However, if you refresh the page or close and expand the *Pipelines* section, the *Name* filter is applied. The same behavior is seen when you remove the *Name* filter. This will be resolved in a future release. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1901207[BZ#1901207]).
 
+* Currently, on clusters with IPsec enabled, RHEL 7.9 nodes cannot communicate with RHCOS nodes. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1925925[BZ#1925925])
```

Comment:
Addresses https://github.com/openshift/openshift-docs/issues/26801#issuecomment-775262883#29619
@anuragthehatter

Reply:
s/RHEL/{op-system-base-full}
s/RHCOS/{op-system-first}

These attributes are part of the common-attributes.adoc file